bridge_adaptivity/config/settings/test.py

```diff
@@ -3,16 +3,20 @@
 
 TEST_RUNNER = 'config.test_runner.PytestTestRunner'
 
-DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'NAME': 'travicidb',
-        'USER': 'test',
-        'PASSWORD': 'password',
-        'HOST': 'localhost',
-        'PORT': '5432',
+try:
```

Comment:
I would use the same construction as we have in local.py: 
```
try:
    import secure
except ImportError:
    import secure_example as secure
```
And extend default database with the `TEST` block see: https://docs.djangoproject.com/en/2.0/ref/settings/#std:setting-DATABASE-TEST